pynamodb/models.py

```diff
@@ -1177,6 +1182,17 @@ def _get_indexes(cls):
                     cls._indexes[pythonic(LOCAL_SECONDARY_INDEXES)].append(idx)
         return cls._indexes
 
+    def _set_ttl_field(self):
+        """
+        Sets the TTL field specified in the Meta if the field is not set.
```

Comment:
To be clear, this sets the field if it is not set. So enabling this on a model and performing a read-modify-write on all records would set the TTL for all records?

Reply:
Maybe call it "set_default" to mirror the existing `setdefault` pattern in Python?

Reply:
Correct.. The plan is to correctly set the TTLs for existing records before doing this (Although this can be quite difficult..)